Codigo/Protocol.bas

```diff
@@ -13083,7 +13083,7 @@ Private Sub HandleCreateItem(ByVal UserIndex As Integer)
             End If
         
             ' Lo registro en los logs.
-            If (.flags.Privilegios And (e_PlayerType.Admin Or e_PlayerType.Dios)) <> 0 Then
+            If (.flags.Privilegios And (e_PlayerType.Admin)) <> 0 Then
```

Comment:
creo que seria `And (e_PlayerType.Admin <> 0) Then`

Reply:
Ok pruebo eso